homeassistant/components/tplink/light.py

```diff
@@ -141,27 +145,29 @@ class LightFeatures(NamedTuple):
     alias: str
     model: str
     supported_features: int
-    min_mireds: float
-    max_mireds: float
+    min_mireds: int
+    max_mireds: int
     has_emeter: bool
 
 
-class TPLinkSmartBulb(LightEntity):
+class TPLinkSmartBulb(TPLinkEntity, LightEntity):
     """Representation of a TPLink Smart Bulb."""
 
     def __init__(self, smartbulb: SmartBulb) -> None:
         """Initialize the bulb."""
+        super().__init__(smartbulb)
         self.smartbulb = smartbulb
-        self._light_features = cast(LightFeatures, None)
-        self._light_state = cast(LightState, None)
+        self._host: str = self.smartbulb.host
+        self._light_features = TPLinkSmartBulb._get_light_features(smartbulb)
```

Comment:
We can still use `self` to access static methods.
```suggestion
        self._light_features = self._get_light_features(smartbulb)
```